hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/utils/shims/Hive2Shim.java

```diff
@@ -42,6 +42,11 @@ public Writable getTimestampWriteable(long value, boolean timestampMillis) {
     return new TimestampWritable(timestamp);
   }
 
+  @Override
+  public Object getPrimitiveJavaObject(Object o) {
```

Comment:
Maybe we can name it `unwrapTimestampAsPrimitive`

Reply:
Done